templates/ts/sso-tab-with-obo-flow/package.json.tpl

```diff
@@ -10,6 +10,8 @@
     "@microsoft/teams-js": "^2.22.0",
     "@microsoft/teamsfx": "^2.2.0",
     "@microsoft/teamsfx-react": "^3.0.0",
+    "ajv": "^8.17.1",
```

Comment:
Why we need those two dependencies?

Reply:
During my local debug, not including this dependency results errors. The detailed reason is writed in the description.

Reply:
What puzzles me is that no one else has reported this error. Is it possible that it only occurs on my local debug?

Reply:
I think it's a bad practice to add any dependency that is not required in our source code.

Reply:
Okay, I'll remove these dependency.